script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_6_2.txt

```diff
@@ -136,42 +136,42 @@ His mastery of the billhook is also lost.
 The vampire swings his billhook haphazardly as if to shake off the fear in front of him. But Arcueid is not going to let that disgraceful opening pass by.
 <Page38>
 "It's cold――― It's cold――― This place, is cold―――!"
-The vampire jumps as he gives off a scream.
-With the last of his blood, Vlov jumps to the back. No, he jumps above.
+The vampire leaps as he gives off a scream.
+With the last of his blood, Vlov jumps backwards. No, upwards.
 <Page39>
-He jumps from the lower ground and towards the surface.
+He jumps from the lower ground towards the surface.
 Even a height of 40 meters is not enough to stop a vampire.
 It's likely that he is planning to replenish his blood supply on the surface.
 <Page40>
-The green ivy tangles around his legs.
-The ivy that sprouted from beneath the rubble grows at a terrifying speed, grabbing Vlov's feet and pulling him back into the lower ground.
+Green ivy tangles around his legs.
+The ivy that sprouted from beneath the rubble grows at a terrifying speed, grabbing Vlov's feet and pulling him back down.
 <Page41>
 "A―――AA, AAAA―――!"
 A river of flame appears once again at Vlov's feet.
-Surrounded by a fire that burns only life, the green ivy gets reduced to ashes.
+Surrounded by a fire that burns only life, the green ivy is reduced to ashes.
 However, the shackles do not vanish.
-New ivy uses the ashes as a nursery and entangles itself with the chains.
+New ivy emerges from the ashes and is entwined with the chains.
 <Page42>
 No matter how many times they burn, the shackles cannot be removed. They continue to loop like a phoenix.
 That's right. If vampires are considered immortal, then the workings of nature are also an eternal, infinite cycle.
 <Page43>
 That ivy is a manifestation of that. The only way to escape is with a fire that burns this entire planet to the core.
-But for Vlov――― No, for a 'curse' of a vampire-like that, Arcueid's breath alone is more than enough to extinguish it.....!
+But for Vlov――― No, for a 'curse' of a vampire like him, Arcueid's breath alone is more than enough to extinguish it.....!
 <Page44>
 ―――Arcueid runs.
-From the intensity of her stance, I can tell that this is the final blow.
+From the intensity of her stance, I can tell that this is the finishing blow.
 <Page45>
-All of Vlov's counterattacks are destroyed.
+All of Vlov's counterattacks have been destroyed.
 With this, she decides that this is the end of the battle.
 I reach the same conclusion.
 There is no need to wait for my assistance. At this rate, Arcueid will crush Vlov into pieces.
 <Page46>
 ...And yet.
-And yet I can't help but feel a bad premonition.
+And yet I can't help but have a bad premonition.
 Something is wrong.
-His agony and his breathing preach out to me that it is not so.
+His agony and breathing preach out that it is not so.
```

Comment:
Can I just flag this 'preach out', this is not a very familiar term. Perhaps.... if I've interpreted this right:

His agony and breathing suggest otherwise.